templates/data/spark/support.html

```diff
@@ -0,0 +1,463 @@
+                    <div class="col-3 col-medium-3">
+                        <h3 class="p-heading--5">Spark data lake platform architecture consulting</h3>
+                    </div>
+                    <div class="col-6 col-medium-3">
+                        <p>
+                            Work with a Canonical platform architect to design the open source data solution that works best for you. Design services covering requirements analysis, environment planning, initial dimensioning and solution design. Fully documented and aligned with your needs.
+                        </p>
+                    </div>
+                </div>
+                <hr class="p-rule--muted" />
+                <div class="row p-section--shallow">
+                    <div class="col-3 col-medium-3">
+                        <h3 class="p-heading--5">Spark data lake platform deployment service</h3>
+                    </div>
+                    <div class="col-6 col-medium-3">
+                        <p>
+                            Canonical engineers deploy Charmed Spark to the cloud or data centre of your choice. Kubernetes and air-gapped environments are fully supported.
+                        </p>
+                        <hr class="p-rule--muted" />
+                        <p>
+                            <a href="https://ubuntu.com/pricing/consulting">Explore pricing&nbsp;&rsaquo;<span class="u-off-screen">explore pricing for consultancy</span></a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <div class="row">
+            <hr class="p-rule" />
+            <div class="col-3 col-medium-6">
+                <h2>Training</h2>
+            </div>
+            <div class="col-3 col-medium-3">
+                <h3 class="p-heading--5">Charmed Spark training</h3>
+            </div>
+            <div class="col-6 col-medium-3">
+                <p>
+                    Get hands-on training with Spark and Charmed Spark, Canonical's solution for Apache Spark, in a three day workshop delivered remotely or on-site.
+                </p>
+                <hr class="p-rule--muted" />
+                <p>
+                    <a class="p-button" href="/contact-us" aria-controls="data-spark-modal">Get in touch<span class="u-off-screen">Get in touch</span></a>
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <hr class="p-rule is-fixed-width" />
+
+    <section class="p-strip is-deep">
+        <div class="u-fixed-width">
+            <p class="p-heading--2">
+                <a href="/contact-us" aria-controls="data-spark-modal">Contact us to get support for Spark&nbsp;&rsaquo;</a>
+            </p>
+        </div>
+    </section>
+
+    <section class="p-section">
+        <div class="row--50-50">
+            <hr class="p-rule" />
+            <div class="col p-section">
```

Comment:
Instead of wrapping each col with `p-section`, can you wrap the row in a singular `p-section`?